.gitignore

```diff
@@ -1 +1,2 @@
 /builddir
+.vscode
```

Comment:
You can hide IDE configuration from git by adding another .gitignore file inside the ".vscode" folder, with "/*" as content. In this way, you hide the folder without modifying the top level .gitignore file.

Reply:
I will try that. Thank you.

Reply:
reverted the changes